simulator/mips/mips_instr.cpp

```diff
@@ -195,8 +195,8 @@ const std::unordered_map <uint8, MIPSInstr::ISAEntry> MIPSInstr::isaMapMIPS32 =
     {0x00, { "madd",  OUT_R_ACCUM,   0, &MIPSInstr::execute_madd,    32} },
     {0x01, { "maddu", OUT_R_ACCUM,   0, &MIPSInstr::execute_maddu,   32} },
     {0x02, { "mul",   OUT_R_ARITHM,  0, &MIPSInstr::execute_mult,    32} },
-    {0x04, { "msub",  OUT_R_DIVMULT, 0, &MIPSInstr::execute_unknown, 32} },
-    {0x05, { "msubu", OUT_R_DIVMULT, 0, &MIPSInstr::execute_unknown, 32} },
+    {0x04, { "msub",  OUT_R_SUBTR,   0, &MIPSInstr::execute_msub,    32} },
```

Comment:
If you use `MIPSInstr::execute_mult`, you may remove these new functions completely